nixos/modules/services/networking/tailscale-derper.nix

```diff
@@ -20,6 +20,15 @@ in
         description = "Domain name under which the derper server is reachable.";
       };
 
+      enableNginx = lib.mkOption {
```

Comment:
We should probably name this option configureNginx to be less ambiguous

Reply:
nice, search "Nginx" in search.nixos.org and get lot of "configureXXX", so "configureNginx" is beter in our PR.

i rename it. thanks!